com.unity.ml-agents/Runtime/Agent.cs

```diff
@@ -119,6 +119,13 @@ internal struct AgentParameters
             public int maxStep;
         }
 
+        public int TeamId {
+            get { return GetComponent<BehaviorParameters>().TeamId; }
+        }
+        public string BehaviorName  {
```

Comment:
This is new API, Is that okay ?

Reply:
Yeah, I think it's fine. Only question is whether we should cache the BehaviorParameters - I believe `GetComponent<>` has non-trivial cost.

Reply:
Ok, I now use the cached BehaviorParameters